crates/uv-cli/src/lib.rs

```diff
@@ -2168,6 +2168,17 @@ pub struct ToolInstallArgs {
 #[allow(clippy::struct_excessive_bools)]
 pub struct ToolListArgs;
 
+#[derive(Args)]
+#[allow(clippy::struct_excessive_bools)]
+pub struct ToolDirArgs {
+    /// Show the directory into which `uv tool` will install executables.
+    ///
+    /// By default, `uv tool dir` shows the directory into which the tool Python environments
+    /// themselves are installed, rather than the directory containing the linked executables.
+    #[arg(long, alias = "executable")]
```

Comment:
Was there a particular motivation for adding an alias?

Reply:
In the moment it felt odd that we use the term "executable" everywhere but `--bin` for the directory. I'll remove it though.

Reply:
Fixed: https://github.com/astral-sh/uv/pull/5187